bin/nuxt-generate

```diff
@@ -159,6 +160,18 @@ if (cluster.isMaster) {
       })
       global.consola.error('==== Error report ==== \n' + report.join('\n\n'))
     }
+
+    const workersWithErrors = Object.values(workerInfo).filter(i => i.code !== 0)
+    if (workersWithErrors.length > 0) {
+      global.consola.fatal(`${workersWithErrors.length} workers failed:\n${util.inspect(workersWithErrors)}`)
```

Comment:
I didn't add this trick to exit on any `consola.fatal()` calls. I agree with pi0 in [this comment](https://github.com/nuxt/nuxt.js/pull/4806/files#r249243316), that we should either way also exit here.

I think it's good practice to not exit in library code, because it makes it very difficult to reuse/test such code, and instead exit only in the CLI code.

Reply:
I've added `--fail-on-error`, but not entirely sure... LMK if I should change it.

I agree that it would be better if the program would exit as soon as the first error happens that would lead to termination. For making such a change I would need to understand the code base better. Since I currently don't use _nuxt-generate-cluster_ (any more) I won't be able to spend that time on the project :(
Maybe you want to merge the change nevertheless, or contribute the adjustment yourself?